src/mixins/view.js

```diff
@@ -215,58 +187,7 @@ export default {
 
   bindUIElements: function() {
```

Comment:
OHHH this is perfect.. composes both of the mixin behaviors together.  This just means that `unbindUIElements` should be pulled out of the ui mixin and put here.